/**:
  ros__parameters:
    robot: ""
    urdf_path: ""
    yml_file_path: ""
    time_dilation_factor: 0.5
    max_attempts: 10
    num_graph_seeds: 6
    num_trajopt_seeds: 6
    include_trajopt_retract_seed: True
    num_trajopt_time_steps: 32
    joint_states_topic: "/joint_states"
    interpolation_dt: 0.025
    collision_cache_cuboid: 20
    collision_cache_mesh: 20
    workspace_file_path: ''
    grid_size_m: [2.0, 2.0, 2.0]
    voxel_size: 0.05
    read_esdf_world: False
    publish_curobo_world_as_voxels: False
    add_ground_plane: False
    publish_voxel_size: 0.05
    max_publish_voxels: 50000
    tool_frame: ""
    grid_center_m: [0.0, 0.0, 0.0]
    esdf_service_name: "/nvblox_node/get_esdf_and_gradient"
    enable_curobo_debug_mode: False
    override_moveit_scaling_factors: False
    update_link_sphere_server: 'planner_attach_object'
    enable_cuda_mps: False
    cuda_mps_pipe_directory: '/workspaces/isaac_ros-dev/ros_ws/mps_pipe_dir'
    cuda_mps_client_priority: 0
    cuda_mps_active_thread_percentage: 100
    moveit_collision_objects_scene_file: ''
